repositories.manifest

```diff
@@ -4,8 +4,8 @@ summary: imgui project repository
 :
 role: prerequisite
 location: https://pkg.cppget.org/1/stable
-#trust: ...
+trust: 70:64:FE:E4:E0:F3:60:F1:B4:51:E1:FA:12:5C:E0:B3:DB:DF:96:33:39:B9:2E:E5:C2:68:63:4C:A6:47:39:43
 
-#:
-#role: prerequisite
-#location: https://git.build2.org/hello/libhello.git
+:
+role: prerequisite
+location:https://github.com/build2-packaging/opengl-meta.git#master
```

Comment:
Is this close to be published as a package?

Reply:
This is a bit off-topic, but to give you some insight:
The package depends on the current staged version of build2, so release to cppget.org will only bet possible as soon as 0.15.0 is released.
Also the package is not well tested. It would require more "examples" to verify that the individual targets export the correct linker flags. Feel free to contribute a few "hello-triangle" examples, e.g. for GL, EGL, GLES, GLU or GLUT and make it work on maybe at least Windows and Linux.

Reply:
Noted on my todo list for future improvements.